socorro/external/es/super_search_fields.py

```diff
@@ -802,6 +802,7 @@ def apply_schema_properties(fields, schema):
             "type": "multi_field",
         },
     },
+    "android_packagename": keyword_field(name="android_packagename"),
```

Comment:
`keyword_field` is a function that generates all the data for a keyword field. There are a few other functions that work like this for other data types. I always intended to go through `super_search_fields.py` and rework `FIELDS` to use the field functions since it's much easier to work with.